src/formatting.ts

```diff
@@ -32,6 +33,13 @@ function format(
         result += record.msg;
       } else if (value === level) {
         result += levelToString(record.level);
+      } else if (value === trace) {
+        const errorStack = new Error().stack ?? '';
```

Comment:
I think we can do something simpler here, if the `stack` is `== null`, then we just don't do anything, and substitute `${trace}` for `''`. So we should do early termination here.